servicecontrol/installation.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Installing ServiceControl
+summary: Installation Options for ServiceControl
+tags:
+- ServiceControl
+- Installation
+- 
+---
+
+The ServiceControl installation fileconsists of an embedded MSI bootstrapper EXE and an embedded MSI.  This installation can be executed standalone or via the Particular Software Platform Installer.

```

Comment:
typo "fileconsists".
is it correct to mention the PI even if it is not supported for production machines?